src/entities/Basket/Basket.validation.ts

```diff
@@ -18,9 +18,10 @@ export const validateBasketStatus = (status: string): ValidationResult => {
         "accepted",
         "rejected",
         "cancelled",
-        "offered",
         "failedByRequestor",
         "failedByTarget",
+        "successByRequestor",
+        "successByTarget",
         "success"
```

Comment:
One point - sucess and pending are statuses that should not be present in the body of the request.. Pending is default after having created the basket - and we never go back do that state.
Success the same - it should be set AUTOMATICALLY (pierwsza osoba zaznacza success, druga osoba zaznacza sukces ze swojej strony  - automatycznie ustawia się sukces 
`if (basket.status === 'successByRequestor' && req.body.status === 'successByTarget' || basket.status === 'successByTarget' && req.body.status === 'successByRequestor) {
await Basket.updateOne({ _id: req.params.id }, { status: 'success' });
    return res.status(OK).send("Basket status updated");
  } catch (error) {
    return res.status(BAD_REQUEST).send(error.message);
  }
}}`

Reply:
For validation of the newly created basket we use `validateBasketReq` and there there is no status allowed. `validateBasketStatus` is used only for the update. Also in `new Basket` there is default status `offered` (need to change it to 'pending'). Actually, this is funny In how many places we have defined a list of possible statuses - I see I need to update the mongoose schema ;) And the second part is added